pkg/registry/core/pod/strategy_test.go

```diff
@@ -2031,3 +2031,117 @@ func Test_mutatePodAffinity(t *testing.T) {
 		})
 	}
 }
+
+func TestPodLifecycleSleepActionEnablement(t *testing.T) {
```

Comment:
how is this test different than the test on drop-disabled?  Do we need both?

Reply:
I refactored both tests in second commit.